modules/doc/content/help/inl/hpc_binary.md

```diff
@@ -0,0 +1,251 @@
+
+   ```bash  style=background-color:#151B54
+   ssh <your hpc user id>@hpclogin
+   ```
+
+## Connect to an INL-HPC Machine
+
+When you login to "hpclogin" you will then need to connect to an HPC machine such as Sawtooth, for
+a complete list of machines please refer to the internal INL HPC website:
+[hpcweb.hpc.inl.gov](https://hpcweb.hpc.inl.gov).
+
+Assuming you want to work on Sawtooth, you can access it by logging into one of the two
+available login nodes: "sawtooth1" or "sawtooth2".
+
+```bash
+ssh sawtooth2
+```
+
+This will log you into the login node of a an HPC machine, you can verify that you connected to
+the machine by displaying the host name:
+
+```bash
+$ echo $HOSTNAME
+sawtooth2
+```
+
+!alert note title=Login nodes should not be used for simulations
+The login nodes, as the name suggests, are simply for accessing the machine and scheduling jobs or
+requesting an interactive job. Please do not run simulations on the login nodes, your account could
+be suspended if this occurs too often.
+
+## Load Application Environment
+
+Next, it is necessary to load the desired application environment. For this example it is
+assumed that BISON is the desired application. Others include "GRIFFIN" and "BLUE_CRAB". The
+correct environment is loaded using the module command as follows.
+
+```bash
+module load use.moose moose-apps BISON
+```
+
+At this point you should be able to run the executable. For this example, the BISON executable
+help can be displayed by running the following command.
+
+```bash
+/apps/herd/bison/bison-opt --help
+```
+
+## Create a Test Input File
+
+To demonstrate the use of the binary and to verify that it is operating correctly, create a
+test input file. For example, the following input file models steady-state diffusion and
+works with any MOOSE-based application.
+
+!listing moose/test/tests/kernels/simple_diffusion/simple_diffusion.i
+
+This file can be created in your INL-HPC home directory or copied via scp from your local machine.
+In both cases, first create a location for the file. For example, the following creates
+a "testing" folder in your remote home directory.
+
+```bash
+mkdir ~/testing
+```
+
+If you want to create the file on the remote machine you will need to use a terminal editor to create
+the file such as "emacs" or "vi". Alternatively, you can create the file on a local machine and copy
+it to the remote, by running the following from your local machine, assuming you created a
+file "input.i" with the aforemention content.
+
+```bash  style=background-color:#151B54;
+scp /path/to/the/local/input.i <your hpc user id>@hpclogin:~/testing/input.i
+```
+
+## Running Test Simulation
+
+### (1) Start-up an interactive job
+
+As mentioned above simulations should not be executed on the login nodes. For this example
+start an interactive job by requesting a single processor for one hour within the "moose"
+project queue.
+
+```bash
+qsub -I -l select=1:ncpus=1 -l walltime=1:0:0 -P moose
```

Comment:
```suggestion
qsub -I -l select=1:ncpus=1 -l walltime=1:00:00 -P moose
```

Reply:
I would use a standard time format for instructional use. I actually checked your syntax to make sure it worked because it looked so funny to me.